docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,638 +1,772 @@
 
-**Figure: Mesh visual**
+The following table lists the supported properties:
 
-![Mesh visual](media/mesh-visual.png)
+**Table: The optional properties of the PrimitiveVisual.**
 
-The following table lists the supported properties. The visual type is `Mesh` and the visual map is `MeshVisual`.
+| Property            | Type                       | Shape to which the property applies  | Description                                                                                                        |
+|---------------------|----------------------------|--------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| `Shape`             | `PrimitiveVisualShapeType` | all                                  | The shape to render. The default value is `Sphere`. For more information, see the [table](#table_primit_possible_values). |
+| `MixColor`          | `Color`                    | all                                  | The color of the shape. The default value is `[0.5f, 0.5f, 0.5f, 1.0f]`.                                           |
+| `Slices`            | `int` in a range 1-255     | all                                  | The number of slices going around the shape. The default value is `128`.                                           |
+| `Stacks`            | `int`                      | all                                  | The number of layers going down the shape. The default value is `128`.                                             |
+| `ScaleTopRadius`    | `float` greater than or equal to 0 | `ConicalFrustrum`                         | The scale of the top circle radius of a conical frustum. The default value is `1.0`.                  |
+| `ScaleBottomRadius` | `float` greater than or equal to 0 | `ConicalFrustrum`<br>`Cone`               | The scale of the bottom circle radius. The default value is `1.5`.                                    |
+| `ScaleHeight`       | `float` greater than or equal to 0 | `ConicalFrustrum`<br>`Cone`<br>`Cylinder` | The height scale of the conic. The default value is `3.0`.                                            |
+| `ScaleRadius`       | `float` greater than or equal to 0 | `Cylinder`                                | The radius scale of the cylinder. The default value is `1.0`.                                         |
+| `ScaleDimensions`   | `Vector3`                  | `Cube`<br>`Octahedron`<br>`BevelledCube`          | The dimensions of the cuboid. Scales in the same way as for a 9-patch image. The default value is `[1, 1, 1]`.     |
+| `BevelPercentage`   | `float` in a range 0-1     | `BevelledCube`                       | Specifies how bevelled the cuboid should be, based on the smallest dimension. It affects the ratio of the outer face widths to the width of the overall cube. The default value is `0.0`, which specifies no bevel. |
```

Comment:
Can you please elaborate on this :
It affects the ratio of the outer face widths to the width of the overall cube.

Reply:
please check now